src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -52,6 +52,31 @@ public static JsonSerializerOptions Default
 
         private static JsonSerializerOptions? s_defaultOptions;
 
+        /// <summary>
+        /// Gets a read-only, singleton instance of <see cref="JsonSerializerOptions" /> that uses the web configuration.
+        /// </summary>
+        /// <remarks>
+        /// Each <see cref="JsonSerializerOptions" /> instance encapsulates its own serialization metadata caches,
+        /// so using fresh default instances every time one is needed can result in redundant recomputation of converters.
+        /// This property provides a shared instance that can be consumed by any number of components without necessitating any converter recomputation.
+        /// </remarks>
+        public static JsonSerializerOptions Web
+        {
+            [RequiresUnreferencedCode(JsonSerializer.SerializationUnreferencedCodeMessage)]
+            [RequiresDynamicCode(JsonSerializer.SerializationRequiresDynamicCodeMessage)]
+            get
+            {
+                if (s_webOptions is not JsonSerializerOptions options)
+                {
+                    options = GetOrCreateWebOptionsInstance();
+                }
+
+                return options;
```

Comment:
This could be simplified:
```suggestion
                return s_webOptions is JsonSerializerOptions options : GetOrCreateWebOptionsInstance();
```

If you could also update the existing method to follow the same style that would be great 👍